### PR TITLE
Update images.md

### PR DESCRIPTION
**Pull request by esbc-disciple**
include explicit instructions on pulling arm64-based images

### CODE REVIEW

File: docs/operations/images.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Images
 
-As of Kubernetes 1.27 the default images used by kOps are the **[official Ubuntu 22.04](#ubuntu-2204-jammy)** images.
+As of Kubernetes 1.27 the default images used by kOps are the x86_64 (amd64) **[official Ubuntu 22.04](#ubuntu-2204-jammy)** images. If you want to pull the default kOps image, but for an arm64 processor architecture, you can specify an arm64-based instance name in the --control-plane-size and --node-size parameters of the kops create cluster command.
```
Comment (hakman) [inline docs/operations/images.md:3]: The topic here is how to find the right images and I don't think this change adds more clarity.

File: docs/operations/images.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
-You can choose a different image for an instance group by editing it with `kops edit ig nodes`.
+You can choose a different image for an instance group by editing it with `kops edit ig nodes`.
```
Comment (hakman) [inline docs/operations/images.md:5]: Please revert.
```suggestion
You can choose a different image for an instance group by editing it with `kops edit ig nodes`.
```

**Review by hakman — CHANGES_REQUESTED** [review]
Hi @esbc-disciple, thank you for trying to add more info on how to use ARM64. Nice to see such initiative. I added a few comments. Please take a look.

File: docs/operations/images.md (hunk 1 above)
Comment (esbc-disciple) [reply docs/operations/images.md:3]: I agree, especially in light of the edits you made to commands, adding a wildcard in the proc-architecture position of the search string. 

My thought process in adding this note was to try to help folks bring up their ARM64 instances without editing config (which is super helpful when you are just getting started and learning). I am OK with it being nixed, nonetheless. With the edit to the search strings, folks should be able to read between the lines on how to instantiate ARM images.

File: docs/operations/images.md [hunk 3]
```diff
@@ -84,6 +84,15 @@ aws ec2 describe-images --region us-east-1 --output table \
   --filters "Name=name,Values=al2023-ami-2*-kernel-6.1-x86_64"
```
[the remark this answers is not in the document]
Comment (esbc-disciple) [reply docs/operations/images.md]: Yes. This is a Homer Simpson d'oh moment on my part! The way you suggested is much cleaner.

**Comment by esbc-disciple** [comment]
@hakman,

I committed the changes we discussed. Let me know if I need to do anything else. Thank you for making it easy to suggest changes!

**Review by hakman — APPROVED** [review]
Thanks!

**Pull request merged**